LogMonitor/src/LogMonitor/EtwMonitor.cpp

```diff
@@ -1277,7 +1277,8 @@ EtwMonitor::GetPropertyLength(
             // EVENT_PROPERTY_INFO.length field will be zero.
             //
             if (TDH_INTYPE_BINARY == EventInfo->EventPropertyInfoArray[Index].nonStructType.InType &&
-                TDH_OUTTYPE_IPV6 == EventInfo->EventPropertyInfoArray[Index].nonStructType.OutType)
+                TDH_OUTTYPE_IPV6 == EventInfo->EventPropertyInfoArray[Index].nonStructType.OutType &&
+                EventInfo->EventPropertyInfoArray[Index].length == 0)

```

Comment:
This fix addition is motivated by this read: https://learn.microsoft.com/en-us/windows/win32/api/tdh/nf-tdh-tdhformatproperty#:~:text=The%20InType%20member%20of%20the%20EVENT_PROPERTY_INFO%20structure%20is,Length%20member%20of%20the%20EVENT_PROPERTY_INFO%20structure%20is%200